pkg/pillar/types/verifiertypes.go

```diff
@@ -85,6 +85,45 @@ func (status VerifyImageStatus) VerifyFilename(fileName string) bool {
 	return ret
 }
 
+// ImageDownloadDirNames - Returns pendingDirname, verifierDirname, verifiedDirname
+// for the image.
+func (status VerifyImageStatus) ImageDownloadDirNames() (string, string, string) {
+	downloadDirname := DownloadDirname + "/" + status.ObjType
+
+	var pendingDirname, verifierDirname, verifiedDirname string
+	if status.IsContainer {
+		pendingDirname = downloadDirname + "/pending/" + status.ImageID.String()
+		verifierDirname = downloadDirname + "/verifier/" + status.ImageID.String()
+		verifiedDirname = downloadDirname + "/verified/" + status.ImageID.String()
+	} else {
+		// Else..VMs
+		pendingDirname = downloadDirname + "/pending/" + status.ImageSha256
+		verifierDirname = downloadDirname + "/verifier/" + status.ImageSha256
+		verifiedDirname = downloadDirname + "/verified/" + status.ImageSha256
+	}
+	return pendingDirname, verifierDirname, verifiedDirname
+}
+
+// ImageFilenames - Returns pendingFilename, verifierFilename, verifiedFilename
+// for the image
+func (status VerifyImageStatus) ImageFilenames() (string, string, string) {
```

Comment:
Either both functions should have "Download" in their names, or neither should have it.
They return the directory vs filename for the same thing hence that difference in naming is confusing.

Reply:
Ah... Good catch.. Fixed it..